catppuccin_tmux.conf

```diff
@@ -188,7 +188,7 @@ set -ogqF @catppuccin_window_current_right_separator "#{@catppuccin_window_right
       "#{E:@_ctp_w_text_style}#{E:@catppuccin_window_left_separator}#{E:@_ctp_w_text_style}#{@catppuccin_window_text}#{@_ctp_w_flags}"
     set -agF window-status-format "#{E:@catppuccin_window_middle_separator}"
     set -agF window-status-format \
-      "#{E:@_ctp_w_number_style} #{@catppuccin_window_number}#{E:@catppuccin_window_right_separator}"
+      "#{E:@_ctp_w_number_style}#{@catppuccin_window_number}#{E:@catppuccin_window_right_separator}"
```

Comment:
Please change the default value of `catppuccin_window_number` so that this is a seamless change for users

Reply:
Updated [here](https://github.com/catppuccin/tmux/pull/463/commits/368a0ed985cd2cd5b6970321fa7a9477cc31703e)

Reply:
Thanks! Can you update the documentation as well?

Reply:
What kind of update did you have in mind for this?

Reply:
Add a space to the config reference